src/NServiceBus.AcceptanceTests/NServiceBus.AcceptanceTests.csproj

```diff
@@ -20,6 +20,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" PrivateAssets="All" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" PrivateAssets="All" />
+    <PackageReference Include="Particular.Approvals" Version="0.2.0" />
```

Comment:
This needs `PrivateAssets="All"` because we build a source package from the project.

Reply:
Fixed